infra/variables.tf

```diff
@@ -1,3 +1,7 @@
 variable "file_hash" {
   type = string
 }
+
+variable "DYNAMODB_TABLE_NAME" {
```

Comment:
use lower case for tf variables

Reply:
also You need a way to pass to this variable a value

for this create a file with .tfvars or auto.tfvars file extension and learn how they work :)